app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    old_file_name = command.split(" ")[1]
+    new_file_name = command.split(" ")[2]
```

Comment:
Don't use `command.split` twice. You should create a separate variable for storing result of `command.split` and use it where you need